docs/DeveloperGuide.md

```diff
@@ -322,46 +322,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 (For all use cases below, the **System** is the `Rapportbook` and the **Actor** is the `user`, unless specified otherwise)
 #### Use case: List contacts
 1. User requests to list contacts.
-2. Rapportbook shows a list of contacts.
-	Use case ends.
+2. Rapportbook shows a list of contacts.  
```

Comment:
Not sure why there's extra spacings at the end of the line, same for a few other lines within this file.

Reply:
The double space is added for creating a new line in mark down.

Reply:
Alright, tho I believe having another blank line before the next line that you want to put into a new line would be neater, i.e.

```
2. Rapportbook shows a list of contacts.

Use case ends.
```